R/class-plotdata.R

```diff
@@ -192,13 +195,21 @@ newPlotdata <- function(.dt = data.table(),
   completeCasesAxesVars <- jsonlite::unbox(nrow(.dt[complete.cases(.dt[, c(x,y), with=FALSE]),]))
   veupathUtils::logWithTime('Determined total number of complete cases across axes and strata vars.', verbose)
 
-  if (evilMode) {
+  if (isEvil) {
     # Assign NA strata values to 'No data', with the exception of continuous overlays which should stay numeric
+    # TODO at some point i want to make sure our logic here is solid. 
+    # is it possible for a continuous overlay to represent something other than color for ex? and what then?
```

Comment:
Maybe it will depend on what we put in the 'overlay' vs 'z variable' bins. For example, would we count size of points in a bubble plot a second overlay or z axis? The correllegram is basically the same as well.